skimage/segmentation/tests/test_random_walker.py

```diff
@@ -15,10 +15,8 @@
 PYAMG_MISSING_WARNING = r'pyamg|\A\Z'
 PYAMG_OR_SCIPY_WARNING = SCIPY_RANK_WARNING + '|' + PYAMG_MISSING_WARNING
 
-if (Version(np.__version__) >= '1.15.0'):
-    NUMPY_MATRIX_WARNING = 'matrix subclass'
-else:
-    NUMPY_MATRIX_WARNING = None
+# Certain version of numpy add this matrix warning, and others remove it.
+NUMPY_MATRIX_WARNING = r'matrix subclass|\A\Z'
```

Comment:
Why are you changing this in this PR?

Reply:
Trying to get green again. Will rollback.

Reply:
The value of these maintenance PRs goes WAYYYYYYYYYYYY down if you don't want multiple fixes to come in the same PR.

Individually, these PRs are low priority, but the value they give to contributors is much higher if you consider them to be part of "getting automated tests to pass again". I ask that you reconsider your request to have seperated

Reply:
Are you advocating for the VW approach to testing? ;)

I'm happy to have big bundled "let's fix everything" PRs. But every change needs to be the right change, not just masking all errors and warnings.